pkg/omf/functions/compat/refresh.fish

```diff
@@ -0,0 +1,14 @@
+function refresh -d "(deprecated) Refresh fish session by replacing current process"
+  printf 'warning: function %srefresh%s is deprecated and will be removed soon.\n %s\n' \
+  (set_color -u) (set_color normal) \
+  (status -t)[5] >&2
```

Comment:
kk. if i call it from a function defined in a file, it works. let's `grep -v` to keep it from talking about standard input and call it a day?